content/master/concepts/providers.md

```diff
@@ -113,28 +113,112 @@ Crossplane installs packages from a local package cache. By
 default the Crossplane package cache is an 
 [emptyDir volume](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). 
 
-Configure Crossplane to use a 
-[PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
-to use a storage location containing the Provider image. Read more about
-configuring the Crossplane Pod settings in the 
-[Crossplane install documentation]({{<ref "../software/install#customize-the-crossplane-helm-chart">}}).
-
-Provide the name of the Provider's `.xpkg` file and set 
-{{<hover label="offline" line="7">}}packagePullPolicy: Never{{</hover>}}.
-
-For example, to install a locally stored version of Provider AWS set the 
-{{<hover label="offline" line="6">}}package{{</hover>}} to the local filename
-and set the Provider's 
-{{<hover label="offline" line="7">}}packagePullPolicy: Never{{</hover>}}.
-
-```yaml {label="offline"}
+To configure Crossplane for offline installation:
+
+1. **Understand `emptyDir` Volume:**
+   The Crossplane package cache is, by default, stored in an [`emptyDir` volume](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). An `emptyDir` is a temporary storage volume created in a pod's memory or on a node's disk. It's initially empty and can be used to share files between containers in the same pod.
+
+2. **Configure with PersistentVolumeClaim (PVC):**
+   Follow the [Crossplane install documentation]({{<ref "../software/install#customize-the-crossplane-helm-chart">}}) for configuring Crossplane to use a PersistentVolumeClaim (PVC) for a storage location containing the Provider image.
+
+3. **Configure Pull Policy for Offline Installation:**
+   In your Provider CR, ensure you set the `package` to the local filename, and set `packagePullPolicy: Never`:
+   
+   ```yaml
+   apiVersion: pkg.crossplane.io/v1
+   kind: Provider
+   metadata:
+     name: offline-provider-aws
+   spec:
+     package: provider-aws
+     packagePullPolicy: Never
+   
+4. **Troubleshooting Tips:**
+
+   Issue: Crossplane not recognizing package changes
+   Solution: ensure the correct package is mounted and accessible by Crossplane. Check file permissions and verify the path configuration.
+
+    
+**Example: configuring Crossplane with a PersistentVolumeClaim (PVC):**
+Follow this example to configure Crossplane using a PersistentVolumeClaim (PVC) for a storage location containing the Provider image.
+
+```shell
+CACHE_PATH="localpackage-cache"
+
+# Extract package and move it to the cache path
+up xpkg xp-extract xpkg.upbound.io/upbound/provider-aws:v0.44.0
+mv out.gz "${CACHE_PATH}/provider-aws.gz"
+chmod 644 "${CACHE_PATH}/provider-aws.gz"
+docker tag xpkg.upbound.io/upbound/provider-aws:v0.44.0 provider-aws
```

Comment:
`up xpkg xp-extract` is a hidden command (the equiv is an alpha command) is it possible to use the `crossplane` CLI to do this? 

Does the package have to be extracted?

Reply:
_bump_

Reply:
Yes, the package does need to be extracted.
For Crossplane offline installation, with ‘–set packageCache.pvc=’ option, The extracted packages must be placed in the directory where the PVC is mounted for Crossplane’s inner registry to recognize them.

Regarding the Crossplane CLI command you mentioned, Could you please provide me with the relevant documentation or details about the command? I’ll review it to see if we can replace the up xpkg xp-extract command with a Crossplane CLI equivalent.

Reply:
Documentation for the Crossplane CLI is [here](https://docs.crossplane.io/v1.14/cli/command-reference/). 

I'm not sure if `extract` is an option (even as a hidden command) but can you check to see if the options match what you're doing here? I'd prefer to have the Crossplane CLI in this doc but if Crossplane doesn't provide it and we have to reference the `up` CLI that's okay.